<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModLibararyClass</class>
 <widget class="QMainWindow" name="ModLibararyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mod Library</string>
  </property>
  <property name="windowIcon">
   <iconset resource="modlibrary.qrc">
    <normaloff>:/ModLibrary/icon.ico</normaloff>:/ModLibrary/icon.ico</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="3" column="0">
       <widget class="QPushButton" name="doSearch">
        <property name="text">
         <string>Find</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>230</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>230</height>
         </size>
        </property>
        <property name="title">
         <string>Find</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>231</width>
           <height>216</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="findWhat"/>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findFilename">
            <property name="text">
             <string>File Name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findTitle">
            <property name="text">
             <string>Title</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findArtist">
            <property name="text">
             <string>Artist</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findSampleText">
            <property name="text">
             <string>Sample Texts</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findInstrumentText">
            <property name="text">
             <string>Instrument Texts</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findComments">
            <property name="text">
             <string>Comments</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBoxEx" name="findPersonal">
            <property name="text">
             <string>Personal Comments</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Melody</string>
        </property>
        <widget class="QLineEdit" name="melody">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1" rowspan="4">
       <widget class="QTableWidget" name="resultTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeft|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File Size</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Modified</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeft|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>221</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>221</height>
         </size>
        </property>
        <property name="title">
         <string>Advanced</string>
        </property>
        <widget class="QCheckBox" name="limitSize">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>231</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Limit by File Size</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="limitMinSize">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="limitMaxSize">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
        <widget class="QComboBox" name="limitSizeUnit">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>40</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Byte</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KiB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MiB</string>
          </property>
         </item>
        </widget>
        <widget class="QCheckBox" name="limitFileDate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>231</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Limit by File Date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="limitFileDateMin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="limitFileDateMax">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="limitYear">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>231</width>
           <height>18</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Some module formats offer the possibility to save the last edit date. This is used for guessing the release date.</string>
         </property>
         <property name="text">
          <string>Limit by Guessed Released Date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>140</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="limitTimeMax">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>190</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="limitTime">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>231</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Limit by Song Length (Seconds)</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="limitTimeMin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>190</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="limitReleaseDateMin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="limitReleaseDateMax">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFile"/>
   <addaction name="actionAddFolder"/>
   <addaction name="separator"/>
   <addaction name="actionMaintain"/>
   <addaction name="actionShow"/>
   <addaction name="actionExportPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddFile">
   <property name="text">
    <string>Add &amp;File...</string>
   </property>
  </action>
  <action name="actionMaintain">
   <property name="text">
    <string>&amp;Maintain...</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>&amp;Show All...</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="text">
    <string>&amp;Add Folder...</string>
   </property>
  </action>
  <action name="actionExportPlaylist">
   <property name="text">
    <string>&amp;Export Playlist</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCheckBoxEx</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxex.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>findWhat</tabstop>
  <tabstop>findFilename</tabstop>
  <tabstop>findArtist</tabstop>
  <tabstop>findSampleText</tabstop>
  <tabstop>findInstrumentText</tabstop>
  <tabstop>findComments</tabstop>
  <tabstop>findPersonal</tabstop>
  <tabstop>limitSize</tabstop>
  <tabstop>limitMinSize</tabstop>
  <tabstop>limitMaxSize</tabstop>
  <tabstop>limitSizeUnit</tabstop>
  <tabstop>limitFileDate</tabstop>
  <tabstop>limitFileDateMin</tabstop>
  <tabstop>limitFileDateMax</tabstop>
  <tabstop>limitYear</tabstop>
  <tabstop>limitReleaseDateMin</tabstop>
  <tabstop>limitReleaseDateMax</tabstop>
  <tabstop>limitTime</tabstop>
  <tabstop>limitTimeMin</tabstop>
  <tabstop>limitTimeMax</tabstop>
  <tabstop>melody</tabstop>
  <tabstop>doSearch</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources>
  <include location="modlibrary.qrc"/>
 </resources>
 <connections/>
</ui>
